src/index.ts

```diff
@@ -33,10 +33,12 @@ export = class OutputDcatAp201 {
       const { dcatStream } = getDataStreamDcatAp201(feedTemplate, feedTemplateTransformsDcatAp);
 
       const datasetStream = await this.getDatasetStream(req);
-
-      datasetStream
-        .pipe(dcatStream)
-        .pipe(res);
+      
+      datasetStream.on('error', (err) => {
+        if (req.next) {
+          req.next(err);
+        }
+      }).pipe(dcatStream).pipe(res);
```

Comment:
Don't we also need to do this for the RSS plugin and the CSV plugin?

Reply:
Good point! We do, I have PR up for that:

 https://github.com/koopjs/koop-output-csv/pull/10

https://github.com/koopjs/koop-output-rss/pull/9